spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -111,14 +137,20 @@ def non_enum_attribute
 
     def record_with_array_values
       _enum_attribute = enum_attribute
-      define_model :record_with_array_values do
+      define_model(
+        :record_with_array_values,
+        _enum_attribute => { type: :integer },
+      ) do
```

Comment:
`define_model` now takes a block argument which is the model you're `class_eval`'ing. What that means is that you can get rid of `_enum_attribute = enum_attribute` to make it a little easier:

``` ruby
define_method(
  :record_with_hash_values,
  enum_attribute => { type: :integer }
) do |model|
  model.enum enum_attribute => { active: 0, archived: 1 }
end
```